readthedocs/projects/views/private.py

```diff
@@ -80,9 +81,24 @@ class ProjectDashboard(PrivateViewMixin, ListView):
     model = Project
     template_name = 'projects/project_dashboard.html'
 
+    def email_notification(self, request):
+        email_qs = request.user.emailaddress_set.filter(primary=True)
+        email = email_qs.first()
+        if email:
+            if not email.verified:
+                email_confirm_notification(user=request.user, success=False)
+        else:
+            email_confirm_notification(user=request.user, success=False)
```

Comment:
Why are you making this a celery task?

We are not sending any email or making anything that takes too much time and needs to be async, I think.

We could just write the two lines from the task here.

Reply:
I am not sure about how to send the notifications, saw an example (Webhook notification) and so followed the trend.
Thansk for the review.
I have removed the task.

Reply:
Also, please, be sure to make some manual QA by yourself together with the tests.

Reply:
OK, Thanks.
I will make sure that this won't happen again.